src/System.Reflection.Metadata/tests/PortableExecutable/PEBuilderTests.cs

```diff
@@ -86,6 +95,23 @@ private static unsafe void VerifyStrongNameSignatureDirectory(PEReader peReader,
             peBlob.WriteContentTo(peStream);
         }
 
+        public static IEnumerable<object[]> AllMachineTypes()
```

Comment:
```=> Enum.GetValues(typeof(Machine)).Select(m => (object)m);```

Reply:
Does it need to be weakly typed?

Reply:
Yes, and it actually need to be returned as a enumerator of object[]. The updated code is

return ((Machine[])Enum.GetValues(typeof(Machine))).Select(m => new object[]{(object)m});